clients/rospy/src/rospy/topics.py

```diff
@@ -189,7 +193,7 @@ class Poller(object):
     """
     def __init__(self):
         try:
-            self.poller = select.poll()
+            self.poller = select.epoll()
```

Comment:
How does this affect cross platform compatibility? The [documentation](https://docs.python.org/2/library/select.html#select.epoll) only mentions Linux, what about OS X and Windows?